packages/core/useWebSocket/index.ts

```diff
@@ -113,10 +113,12 @@ function resolveNestedOptions<T>(options: T | true): T {
  *
  * @see https://vueuse.org/useWebSocket
  * @param url
+ * @param protocols
  */
 export function useWebSocket<Data = any>(
   url: string,
   options: WebSocketOptions = {},
+  protocols: string[] = [],
```

Comment:
I would like to have this part of the options for future extensibility

Reply:
Thought about that. If you think that the WebSocket Class will be extended in the future, I add it to the options.

Reply:
Moved to WebSocketOptions